packages/babel-parser/test/fixtures/typescript/cast/unparenthesized-type-assertion-and-assign/input.ts

```diff
@@ -0,0 +1 @@
+<string>foo = '100';
```

Comment:
Can you also add a test for `(<string> foo) = "100"`? (unless we already have it)

Reply:
We have it https://github.com/babel/babel/blob/7bc72bb451e71d7d8964efdf4438d5a907aae655/packages/babel-parser/test/fixtures/typescript/cast/parenthesized-assert-and-assign/input.ts#L1

Reply:
I meant with the `TSTypeAssertion` syntax, not with `TSAsExpression`

Reply:
Ops wrong file🤦 https://github.com/babel/babel/blob/main/packages/babel-parser/test/fixtures/typescript/cast/parenthesized-type-assertion-and-assign/input.ts

Reply:
Oh ok 👍